easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb

```diff
@@ -0,0 +1,22 @@
+name = 'pgi'
+version = '15.7'
+year = '2015'
```

Comment:
only used in one place, so no reason to use a separate variable?

does it make sense to make the version `2015-15.7`?